webpack.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  entry: {
+    rollbar: './src/bundles/rollbar.js',
+    'rollbar.nojson': './src/bundles/rollbar.nojson.js',
+    'rollbar.umd': './src/bundles/rollbar.umd.js'
```

Comment:
I think customers will want a version of the UMD without JSON and one with. 

I'd expect the build process to add the custom JSON implementation for one build but not the other. The custom JSON code would override `Rollbar._json` or something similar.

e.g.
```js
module.exports = [
    // Rollbar with custom JSON
    {
        name: 'with-json',
        entry: {
            'rollbar': './src/bundles/rollbar.js',
            'json': './src/bundles/json2.js'
        },
        output: {
            library: 'Rollbar',
            path: './dist/bundles/',
            libraryTarger: 'umd.json'
        },
        failOnError: true
    },

    // Rollbar without custom JSON
    {
        name: 'without-json',
        entry: {
            'rollbar': './src/bundles/rollbar.js'
        },
        output: {
            library: 'Rollbar',
            path: './dist/bundles/',
            libraryTarger: 'umd.no-json'
        },
        failOnError: true
    }
];
```